Description: revert upstream change
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-11-28

--- a/read-json.js
+++ b/read-json.js
@@ -4,7 +4,7 @@
 
 var glob = require('glob')
 var normalizeData = require('normalize-package-data')
-var safeJSON = require('json-parse-even-better-errors')
+var safeJSON = require('json-parse-better-errors')
 var util = require('util')
 var normalizePackageBin = require('npm-normalize-package-bin')
 
--- a/test/helpful.js
+++ b/test/helpful.js
@@ -6,7 +6,7 @@
 tap.test('erroneous package data', function (t) {
   readJson(p, function (er, data) {
     t.ok(er instanceof Error)
-    t.ok(er.message.match(/Unexpected token "'" \(0x27\)/))
+    t.ok(er.message.match(/Unexpected token '/))
     t.end()
   })
 })
